### ISSUE TITLE
Link 'x' contains a language reference: undefined

### ISSUE THREAD

**Issue by johnottone** [DavidAnson/markdownlint#656]
This is driving me nuts, and I cannot find a way to ignore this warning. When writing any markdown file, if I include a link or multiple links, as demonstrated below, I get the same warning for each link, and it does not have an error/warning code and I can't find any information, ANYWHERE, on how to ignore it. It is kind of annoying having yellow squiggly lines everywhere on the document. Note, that if I take off the `view=powershell-7.3` the links do not trigger a warning, and I could just do a find replace on all of them, but that's not really a feasible solution. All of the links below are links to Microsoft PowerShell documentation, but this same warning is triggered on any link in markdown, containing similar "URL arguments/parameters". What am I missing? How can I ignore this warning either in the user `settings.json`, or the workspace `settings.json` Please help!!!

```markdown
- Overview
  - [What is PowerShell?](https://learn.microsoft.com/en-us/powershell/scripting/overview?view=powershell-7.3)
  - [What is a PowerShell command?](https://learn.microsoft.com/en-us/powershell/scripting/powershell-commands?view=powershell-7.3)
  - [Discover PowerShell](https://learn.microsoft.com/en-us/powershell/scripting/discover-powershell?view=powershell-7.3)
```

![vscode-markdown-error-20221127-01](https://user-images.githubusercontent.com/112649342/204194600-add5ac3f-1bc0-4dfc-8c52-3d4428159341.png)

**Comment by DavidAnson** [comment]
Those messages are not coming from this extension. Based on a GitHub search, something called linkcheckermd seems most likely: https://github.com/search?q=%22contains+a+language+reference%22&type=code

**Comment by johnottone** [comment]
I was pretty convinced myself that it was not coming from markdownlint, but I figured I would give this a shot, if nothing more than to rule it out. So I have been scouring all my extensions. I looked for LinkCheckerMD, as Microsoft recommends its use, but I do not have it installed. I am still going through my extensions, as we speak. I will disable every one of them if it comes down to that. Thank you so much for your time. At least I can rule out markdownlint as the culprit. It's very strange that I cannot find ANYTHING about this particular error on google or bing, and I am pretty good at finding solutions to obscure problems...Most of the time, anyway...

Thanks again for your time and your support,

John

**Comment by johnottone** [comment]
Now I feel like an idiot. I kept doing a search in my @installed extensions for markdown and lint and link, and LinkCheckerMD never came up. When i just did a direct search for it, it popped up. Apparently it was installed with the Microsoft Authoring Pack and you were 100% correct. LinkCheckerMD was the culprit. Thank you so much.

**Comment by zendu** [comment]
I had the same issue. Searching for `LinkCheckerMD` did not show any extension. I checked for https://github.com/microsoft/linkcheckermd, but I didn't have that extension either. It has same title as Microsoft LinkCheckerMD - "HTTP/s and relative link checker" and identifier "blackmist.linkcheckmd".

HTH.
